sysutils/edid-decode/Portfile

```diff
@@ -0,0 +1,17 @@
+PortSystem       1.0
+
+name             edid-decode
+version          git-e48fb38
```

Comment:
Since there are no actual tagged versions you can use, it's recommended to set version to the datestamp of the commit.

```suggestion
version          20230601
```

Reply:
Fixed (d73a890).